File: Sources/Templates/ProjectTemplates/Benchmark Console/Benchmarks.cs
using System;
using BenchmarkDotNet;
using BenchmarkDotNet.Attributes;

namespace $safeprojectname$
{
    public class Benchmarks
    {
        [Benchmark(Baseline = true)]
        public void Scenario1()
        {
            // Implement your benchmark here
        }

        [Benchmark]
        public void Scenario2()
        {
            // Implement your benchmark here
        }
    }
}

File: Sources/Templates/ProjectTemplates/VB/Benchmark Console/Benchmarks.vb
Imports System
Imports BenchmarkDotNet
Imports BenchmarkDotNet.Attributes

Namespace $safeprojectname$
    Public Class Benchmarks
        <Benchmark(Baseline:=True)>
        Public Sub Scenario1()

            ' Implement your benchmark here

        End Sub

        <Benchmark>
        Public Sub Scenario2()

            ' Implement your benchmark here

        End Sub
    End Class
End Namespace